helm/g2p-sandbox/values.yaml

```diff
@@ -156,10 +178,23 @@ ph-ee-engine:
     image: "419830066942.dkr.ecr.ap-south-1.amazonaws.com/phee-mojaloop"
     imageTag: "latest"
     SPRING_PROFILES_ACTIVE: "bb"
-    hostname: "mojaloop.sandbox.mifos.io"
+    hostname: "mojaloop.sandbox.fynarfin.io"
```

Comment:
keep this as mifos and change from cookbooks

Reply:
changed